tests/virt_autotest/login_console.pm

```diff
@@ -44,19 +45,29 @@ sub double_check_xen_role {
         die 'Double-check xen kernel failed';
     }
 
+    # for modular libvirt, virtxend is expected in "loaded: active or inactive" status.
+    # virtxend.socket seems to be always in "loaded: active" status
+    unless (is_monolithic_libvirtd) {
+        die 'virtxend.socket is not running!' unless script_run("systemctl is-active virtxend.socket") eq 0;
+    }
```

Comment:
Actually this can be further simplified as:
```
die 'virtxend.socket is not running!' if (script_run("systemctl is-active virtxend.socket") ne 0 and !is_monolithic_libvirtd)
```

Reply:
It is too long for one line, so I split them into two lines.